build-scripts/golang-built-rpms/golang-built-rpms

```
@@ -0,0 +1,302 @@
+#!/usr/bin/env bash
+
```

Needs `set -euo pipeline` to prevent silent failure on exception.

---

agreed, this should pretty much come standard (even when hacking up a script...)

---

:man_facepalming: of course ... resolved.